_sources/examples.txt

```diff
@@ -0,0 +1,234 @@
+Examples
+========
+
+The cclib package provides three powerful scripts to parse and write data i.e. ``ccget``, ``ccwrite``, and ``cda``
+
+1. **ccget** is used to parse attribute data from output files.
+2. **ccwrite** has the ability to list out all valid attribute data that can be parsed from an output format. It has the added feature of writing the output file into four different formats i.e. ``json``, ``cjson``, ``cml``, ``xyz``.
+3. **cda** is used for the chemical decomposition analysis of output files.
+
+This page describes how to use the ccget and ccwrite scripts to obtain data from output files.
+
+ccget
+-----
+
+**Input**
+
+The output file used as an example is a Benzeneselenol(C6H5-SeH) molecule optimized using Gaussian09. The input geometry was optimized at the ``B3LYP + 6-311++g(3df,3pd)`` level of calculation.
```

Comment:
No need for pasting an input file here. Perhaps just link to a unit test input file. That is sufficient as an example, I think.